assets/src/edit-story/components/carousel/secondaryMenu.js

```diff
@@ -63,6 +63,10 @@ const Space = styled.span`
 `;
 
 function SecondaryMenu() {
+  const isHelpCenterOpenRef = useRef(false);
+  const isChecklistOpenRef = useRef(false);
+  const isKeyboardShortcutsMenuOpenRef = useRef(false);
```

Comment:
I feel like we could use one ref to track the open state 🤔 

```js
const openPopup = useRef(null);

...
openPopup.current = 'helpCenter'; // or any string
```

Then we have one piece of state where we track this instead of 3 (or more if we add more popups)

Reply:
🤦  yup, thank you!